docs/meetings/dev/2024-10-08

```diff
@@ -0,0 +1,9 @@
+- We talked about how to coordinate interviews with new volunteers and how to schedule calls with them.
+- We talked about the profiles of new volunteers who want to contribute.
+- Discussed security measures for the database
+- Discussed the strong verification, which needs an interface that has not yet been created, Muriel volunteered to create it.
+- Aapeli: he has been very busy this week, so he has been able to progress it as he wanted to
+- Me: (David) Discussed a way to organize tasks and create tickets, so that the coming devs can easily understand what needs to be done, what milestone the task belongs to and possible mockups as figma mockups are often too complex to implement on the first interaction.
+- Ivan: He is progressing with the App, he finished some pending work.
+- Jesse: He has worked on the UX of the map and defined its behavior.
+- Darren: He is looking at a PR from Nicole about a migration of the Material UI components.
```

Comment:
Please also rename the file to include `.md` at the end!

Reply:
🫡